cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -4,9 +4,6 @@
             <a class="link-tab" href="#tab1"><%= gettext("Common Problem Types") %></a>
         </li>
         <li>
-            <a class="link-tab" href="#tab2"><%= gettext("Common Problems with Hints and Feedback") %></a>
```

Comment:
I must have skimmed over this. I was advised to focus on testing requirements and to assume that it had already gone through an engineering review.

That being said, I'm still curious as to how this hooks up with the yaml template. This is just an anchor tag with a translated message.